### PR TITLE
[new release] fiat-p256 (0.2.0)

### PR DESCRIPTION
**Pull request by emillon**
Primitives for Elliptic Curve Cryptography taken from Fiat

- Project page: <a href="https://github.com/mirage/fiat">https://github.com/mirage/fiat</a>
- Documentation: <a href="https://mirage.github.io/fiat/doc">https://mirage.github.io/fiat/doc</a>

##### CHANGES:

*2019-07-23*

### Fixed

- Fix a bug in `generate_key` where it would never actually work when used with
  a proper `rng` function (mirage/fiat#44, @NathanReb)
- Fix benchmark executable. It is now built (but not executed) as part of tests
  (mirage/fiat#45, @emillon)

### Changed

- Use `alcotest` instead of `ppx_expect` for tests (mirage/fiat#43, @emillon)

### CODE REVIEW

File: packages/fiat-p256/fiat-p256.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ ["dune" "build" "-p" name "-j" jobs]
+ ["dune" "runtest" "-p" name] {with-test}
+]
+depends: [
+  "alcotest" {with-test}
+  "asn1-combinators" {with-test}
+  "benchmark" {with-test}
+  "bigarray-compat"
+  "cstruct" {>= "3.5.0"}
+  "dune" {build & >= "1.6.0"}
```
Comment (kit-ty-kate) [inline packages/fiat-p256/fiat-p256.0.2.0/opam]:
```suggestion
  "dune" {>= "1.6.0"}
```
Dune should not be tagged as a build dependency anymore (see https://github.com/ocaml/opam-repository/pull/14266)

File: packages/fiat-p256/fiat-p256.0.2.0/opam [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+]
+homepage: "https://github.com/mirage/fiat"
+bug-reports: "https://github.com/mirage/fiat/issues"
+dev-repo: "git+https://github.com/mirage/fiat.git"
+doc: "https://mirage.github.io/fiat/doc"
+tags: ["org:mirage"]
+build: [
+ ["dune" "subst"] {pinned}
+ ["dune" "build" "-p" name "-j" jobs]
+ ["dune" "runtest" "-p" name] {with-test}
```
Comment (kit-ty-kate) [inline packages/fiat-p256/fiat-p256.0.2.0/opam]:
```suggestion
 ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

**Comment by kit-ty-kate** [comment]
Otherwise it's all good

File: packages/fiat-p256/fiat-p256.0.2.0/opam (hunk 1 above)
Comment (emillon) [reply packages/fiat-p256/fiat-p256.0.2.0/opam]: cool, good to see it's policy now!

**Comment by emillon** [comment]
All refreshed @kit-ty-kate !

**Comment by kit-ty-kate** [comment]
Thanks a lot!

**Pull request merged**